app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -7,9 +7,7 @@ class Api::V2::RepositoriesController < Api::V2::ApiController # rubocop:disable
       generic_repo_wrap_params << option.name
     end
 
-    repo_wrap_params = RootRepository.attribute_names + generic_repo_wrap_params
-
-    wrap_parameters :repository, :include => repo_wrap_params
```

Comment:
These are not necessary, but seems nicer to delegate `atribute_names` call to Rails instead of calling it explicitly here. I'd love to get rid of calling database related stuff during initialization time.